tests/unit/EntityMenu.spec.ts

```diff
@@ -478,6 +478,19 @@ describe('Entity Menu - More actions click tests', () => {
   })
 })
 
+it('renders More actions if historical and digital credential feature is allowed', async () => {
+  businessStore.setState(EntityState.HISTORICAL)
+
+  const wrapper = mount(EntityMenu, {
+    vuetify,
+    mixins: [{ methods: { isAllowed: () => true } }],
+    propsData: { businessId: 'BC1234567' }
+  })
+
+  expect(wrapper.find('.menu-btn').exists()).toBe(true)
+  wrapper.destroy()
+})
+
```

Comment:
I'm interested in seeing a test, "does not render More actions if historical and digital credential feature is not allowed".

Reply:
Would that not be covered by the 'does not render More actions if historical' test?

Reply:
I didn't see that one ;)

Can you update line 456 to "if historical and digital credential feature is not allowed"?